src/test/java/com/google/javascript/cl2dts/nullable.js

```diff
@@ -3,15 +3,15 @@ goog.provide('nullable');
 /** @type {?boolean} */
 nullable.w = true;
 
-//!! Explicitly non-nullable.
+// Explicitly non-nullable.
```

Comment:
I think it's misleading to use the special syntax in a place we don't actually strip anything